### PR TITLE
Restaurant page feature

### PR DESCRIPTION
**Pull request by Addis-Belete**
In this project, I built a basic restaurant page using JavaScript and webpack. All pages are build using DOM manipulations.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello  @Addis0943 👋

**You've done pretty well here fixing, good use of linting rules, nice UI design also. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "restaurant",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```
Comment (mikenath223) [inline package.json]: - [ ] Kindly include the necessary startup scripts:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

File: src/contact.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+function component() {
+  const element = document.getElementById('contact');
+  const header = document.createElement('header');
+  const nav = document.createElement('nav');
+  const home = document.createElement('a');
+  const menu = document.createElement('a');
+  const contact = document.createElement('a');
+  const logo = document.createElement('a');
+  const section = document.createElement('section');
```
Comment (mikenath223) [inline src/contact.js]: - [ ] Please change this and all other regular functions to es6 arrow function syntax.

File: README.md [hunk 3]
```diff
@@ -1,59 +1,70 @@
...
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link](https://addis0943.github.io/Restaurant/)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+## To get local copy up and running follow these simple example steps
```
Comment (mikenath223) [inline README.md:38]: - [ ] Please include instructions to run the project.

**Comment by Addis-Belete** [comment]
Hello...Thank you for your feedback.
I fixed all based on your feedback. kindly review and approve it.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @Addis0943 👋

**You've done pretty well here fixing a few issues raised in the previous review. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/contact.js (hunk 2 above)
Comment (mikenath223) [reply src/contact.js:12]: > * [ ]  Please change this and all other regular functions to es6 arrow function syntax.

- [ ] REQUIRED - As requested previously, please change this function and all other similar regular function syntaxes throughout the code into es6 arrow function syntax. It's important the code keeps up to date with recent trends.

**Comment by mikenath223** [comment]
> * [ ]  Please change this and all other regular functions to es6 arrow function syntax.

- [ ] REQUIRED - As requested previously, please change **all functions** with regular function syntaxes into es6 arrow function syntax. It's important the code keeps up to date with recent trends.

**Comment by Addis-Belete** [comment]
Hello...Thank you for your feedback.
I fixed all based on your feedback. kindly review and approve it.

**Review by GzimAsani — APPROVED** [review]
# STATUS: PROJECT ACCEPTED :green_circle: :partying_face: 

## Hello @Addis0943, thank you for implementing all the required changes :+1: You can merge this pull request and good luck with your next projects :muscle: 

# HAPPY CODING :computer: :desktop_computer:

**Pull request merged**